lib/screens/home_screen.dart

```diff
@@ -93,11 +92,14 @@ class HomeScreenContent extends StatelessWidget {
     final homeViewModel = context.watch<HomeViewModel>();
     return Column(
       children: [
+        if (homeViewModel.currentPage.value != 0) ...[
+          const VerticalSpace(20),
```

Comment:
Should VerticalSpace be .h?

Reply:
If you look into the Vertical space code its value.h. Which means it do that automatically.